server/config.js

```diff
@@ -0,0 +1,2 @@
+ATLAS_URI=mongodb+srv://Cluster13473:teamup@cluster13473.ziewtw1.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
This file should not be committed to our repo - renaming to config.env should make sure it is included in our .gitignore!